README.md

```diff
@@ -90,6 +90,8 @@ Finally, with given scenarios and multi-partner learning approaches, we can addr
          - 20
        gradient_updates_per_pass_count:
          - 8
+       proportion_dataset:
```

Comment:
It seems you wrote `proportion_dataset` in the `README` and `dataset_proportion` in the code!